### ISSUE TITLE
question about the distance information in KITTI dataset

### ISSUE THREAD

**Issue by Jichen66** [harshilpatel312/KITTI-distance-estimation#8]
Hi,

I have a question about  the distance information in KITTI dataset. Can we directly use the value of z location(in camera coordinates) as the ground truth distance between the object and camera?

I am looking for your reply. Thank you very much in advance!

Best Regards

**Comment by SamSamhuns** [comment]
Yes, I think so, I did the training on the raw z location values (without standardizing them) and the prediction works fine. As far as I know, you don't need to standardize the `y` values (z loc). The original author standardizes the z-loc values and later transforms them back which I think is unnecessary.

However, standardizing the `X` values (bbox coordinates) is very beneficial. And I think it is better to use the width and height of the bounding boxes rather than using the four bbox coordinates for the distance regression.

**Comment by Jichen66** [comment]
yes, I also tried the raw z location values before.  Btw, what do you mean by standardizing?

**Comment by SamSamhuns** [comment]
Set the bounding box width and height to 0 mean and 1 standard deviation.

**Comment by ArunMathewML** [comment]
is it necessary to give zloc value for accurate predictions

**Comment by omer-m** [comment]
Hy any can tell me about the dataset... because I can't find it on KITTI - Vision Benchmark Suite website 

http://www.cvlibs.net/datasets/kitti/index.php
